css/style.css

```diff
@@ -6,11 +6,16 @@ body {
 
 h2 {
 	font-size: 40px;
-	margin-bottom: 20px;
+	margin-bottom: 5px;
 }
 
 a {
 	color: #ffc0cb;
+	transition: color 350ms;
+}
+
+a:hover {
+	color: #bfa8fd8f;
```

Comment:
Looks like your hex color here has too many characters. A valid hexadecimal color code should have 3 or 6 characters.